sdk/cognitiveservices/cognitiveservices-inkrecognizer/test/utils/testUtils.ts

```diff
@@ -0,0 +1,29 @@
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+export const isNode =
+  !!process && !!process.version && !!process.versions && !!process.versions.node;
+
+export enum EnvVarKeys {
+  INKRECOGNIZER_API_KEY = "INKRECOGNIZER_API_KEY"
+}
+
+function getEnvVarValue(name: string, addBrowserSuffix?: boolean): string | undefined {
```

Comment:
@KarishmaGhiya I believe you referenced the event hubs or service bus tests for this. We needed different resources for node vs browser there, so we had to do the above.

I dont beleive ink recognizer has the same requirement, so we dont need to add the browser prefix and maintain 2 different variables

Reply:
done